src/Gameplay/ShuffleButton.java

```diff
@@ -0,0 +1,50 @@
+package Gameplay;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class ShuffleButton extends JPanel { //not extending Pile because a button has no need for cards
+
+    private Button shuffleButton;
+    public static String shuffleButtonFileName = "GameplayImages/ShuffleButton", extension = ".png";
```

Comment:
<strike> https://docs.oracle.com/javase/8/docs/technotes/guides/lang/resources.html </strike>

Reply:
<strike> Boring and annoying, but standardised.
Images, sounds, etc. should be located in a dedicated resource part of the package, not in the sources package. </strike>

Reply:
Image properly acquired, using class.resource, weirdly IDE does not separate these on my end.